configure/README.md

```diff
@@ -5,3 +5,30 @@ Scripts used for configuring server nodes
 ## `nginx`
 
 nginx setup. Derived from [`whatwg/misc-server`](https://github.com/whatwg/misc-server)
+
+## `vscode`
+
+1. CTRL+SHIFT+P, search for "preferences open settings (JSON)"
+2. Open the "Workspace settings" tab
+3. Add the property "files.assiciations" if not present or extend it with the value "*.es": "javascript"
+
+Should looks like
+
+```javascript
+    "files.associations": {
+        "*.es": "javascript"
+    }
+```
```

Comment:
EASY PEASY @rianby64. This is great. Now where it should go see above. But wherever it goes this is great.